hbase-balancer/src/test/java/org/apache/hadoop/hbase/master/balancer/StochasticBalancerTestBase.java

```diff
@@ -114,7 +85,13 @@ protected void testWithClusterWithIteration(Map<ServerName, List<RegionInfo>> se
     Map<TableName, Map<ServerName, List<RegionInfo>>> LoadOfAllTable =
       (Map) mockClusterServersWithTables(serverMap);
     List<RegionPlan> plans = loadBalancer.balanceCluster(LoadOfAllTable);
-    assertNotNull("Initial cluster balance should produce plans.", plans);
+    if (plans == null) {
+      LOG.info("First plans are null. Trying more balancer time, or will fail");
```

Comment:
I think that these log messages about relaxing the runtime constraint should be `DEBUG`.